internal/cli/services.go

```diff
@@ -70,13 +70,13 @@ var CmdServiceCreate = &cobra.Command{
 		serviceName := args[1]
 
 		err = client.ServiceCreate(catalogServiceName, serviceName)
-		return errors.Wrap(err, "error creating Epinio Service")
+		return errors.Wrap(err, "error creating Epinio catalog service")
```

Comment:
Catalog service is the input, service is what is created here, if I read the preceding chunk correctly.
```suggestion
		return errors.Wrap(err, "error creating service")
```